uxarray/grid/validation.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from uxarray.io._ugrid import _is_ugrid
+from uxarray.constants import ERROR_TOLERANCE
+
+
+# validation helper functions
+def check_connectivity(self):
```

Comment:
This should be an internal method also
```suggestion
def _check_connectivity(self):
```

Reply:
same comment about the `self`